libs/Cargo.toml

```diff
@@ -14,7 +14,3 @@ resolver = "2"
 
 [workspace.package]
 version = "0.2.7"
-
-[workspace.dependencies]
-uniffi = "0.23.0"
-uniffi_macros = "0.23.0"
```

Comment:
I assume these are for testing. Make sure this change is reverted before merging.